src/components/can_cooperation/src/can_module.cc

```diff
@@ -367,6 +367,7 @@ void CANModule::SendResponseToMobile(application_manager::MessagePtr msg) {
 
 void CANModule::SendTimeoutResponseToMobile(
     application_manager::MessagePtr msg) {
+  LOG4CXX_DEBUG(logger_, "Response to mobile: " << msg->json_message());
```

Comment:
@AKalinich-Luxoft How can we differentiate normal response to mobile and this one? Propose to change log message.